services/worker-manager/src/provider_google.js

```diff
@@ -1,17 +1,60 @@
+const slugid = require('slugid');
+const _ = require('lodash');
 const fs = require('fs');
+const taskcluster = require('taskcluster-client');
+const libUrls = require('taskcluster-lib-urls');
+const uuid = require('uuid');
 const {google} = require('googleapis');
 const {Provider} = require('./provider');
 
 class GoogleProvider extends Provider {
 
-  constructor({id, monitor, notify, project, creds, credsFile}) {
-    super({id, monitor, notify});
+  constructor({
```

Comment:
This class doesn't seem to be implementing `terminate()`, `validate()`, `createResources(workertype)`, `updateResources(workertype)`, `removeResources(workertype)` and `cleanup()`?

Reply:
Yeah, they ended up being unnecessary for this provider. I figured I would leave them in the base class in case other providers want them.